time-series/time-series-api/src/main/java/com/powsybl/timeseries/ast/AbstractMinMaxNodeCalc.java

```diff
@@ -12,30 +12,19 @@
 import com.powsybl.timeseries.TimeSeriesException;
 
 import java.io.IOException;
-import java.util.Objects;
 
 /**
  * @author Geoffroy Jamgotchian {@literal <geoffroy.jamgotchian at rte-france.com>}
  */
-public abstract class AbstractMinMaxNodeCalc implements NodeCalc {
-
-    protected NodeCalc child;
+public abstract class AbstractMinMaxNodeCalc extends AbstractSingleChildNodeCalc {
```

Comment:
Same, you can add AbstractSingleChildNodeCalc in another PR as a refacto that we will merge before.

Reply:
new PR for this modification : #2906

Reply:
merged